pkg/agent/containerd/config_linux.go

```diff
@@ -57,7 +57,7 @@ func setupContainerdConfig(ctx context.Context, cfg *config.Node) error {
 		DisableCgroup:         disableCgroup,
 		IsRunningInUserNS:     isRunningInUserNS,
 		PrivateRegistryConfig: privRegistries.Registry(),
-		ExtraRuntimes:         findNvidiaContainerRuntimes(nil),
+		ExtraRuntimes:         findNvidiaContainerRuntimes(os.DirFS("/")),
```

Comment:
```suggestion
		ExtraRuntimes:         findNvidiaContainerRuntimes(os.DirFS(PathSeparator)),
```

Reply:
Not sure if this really makes sense semantically here, this is also a _linux.go file as well, however I can make this change if you want.

Reply:
@kralicky can you make this change an we can move forward with this PR.

Reply:
Sure thing